### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Summer-Field**
Update READEME.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
...
+
+test-infra provides:
+
+1. GitHub automation in the form of policy enforcement.
+2. chat-ops via /foo style commands.
+3. automatic PR merging.
+
+In this repository, you can configure:
+
+1. Open Source Project's Member Management.
```
Comment (Jeffwan) [inline README.md]: Remove "Open Source"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
 # test-infra
-test-infra configurations
+
+test-infra is a CI/CD system aimming at ByteDance's Open Source Project Management.
+
+test-infra provides:
```
Comment (Jeffwan) [inline README.md]: test-infra provides: -> `## features`

File: README.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
 # test-infra
-test-infra configurations
+
+test-infra is a CI/CD system aimming at ByteDance's Open Source Project Management.
```
Comment (Jeffwan) [inline README.md]: Let's change `Bytedance` to `kitex-contrib` now.

**Review by Jeffwan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Summer-Field) [reply README.md]: OK

File: README.md (hunk 2 above)
Comment (Summer-Field) [reply README.md]: OK

File: README.md (hunk 3 above)
Comment (Summer-Field) [reply README.md]: OK

File: README.md [hunk 4]
```diff
@@ -1,2 +1,14 @@
...
+
+## features
+
+1. GitHub automation in the form of policy enforcement.
+2. chat-ops via /foo style commands.
+3. automatic PR merging.
+
+In this repository, you can configure:
+
+1. Project's Member Management.
```
Comment (Jeffwan) [inline README.md]: let's change from "Project's Member Management." to "Organization's members."

Comment (Summer-Field) [reply README.md]: OK

File: README.md [hunk 5]
```diff
@@ -1,2 +1,14 @@
...
+## features
+
+1. GitHub automation in the form of policy enforcement.
+2. chat-ops via /foo style commands.
+3. automatic PR merging.
+
+In this repository, you can configure:
+
+1. Organization's members.
+2. Project's Label Management.
```
Comment (Jeffwan) [inline README.md]: same here..

**Pull request merged**